### ISSUE TITLE
iDeal purchaseID limit is too short?

### ISSUE THREAD

**Issue by jessevdp** [Fingertips/Ideal#6]
This library limits the `:order_id` (which is mapped to iDeal `purchaseID`) to 12 characters ([code](https://github.com/Fingertips/Ideal/blob/master/lib/ideal/gateway.rb#L372)). I believe that limit has been expanded in later versions of iDeal to 35.

My source:

https://media.rabobank.com/m/756f69006603b3fa/original/iDEAL-Merchant-Integration-Guide_29696264-pdf.pdf

Chapter: 5.2 TransactionRequest

I've tried monkey patching the limit up to `35` in my project, and the iDeal payment seems to go through. (Tested against ING Sandbox.)

Can we up that limit?

**Comment by Manfred** [comment]
Of course! I currently don't have access to any merchant accounts so it would be great if you could create a PR for this change.

**Comment by Manfred** [comment]
One thing to consider is that them library currently only supports version 2 of the protocol so that could be the issue.
